assignment6/rearranging_cars.py

```diff
@@ -0,0 +1,72 @@
+
+class Movement(object):
+    """This class stores information about one movement,
+    i.e. a car moving from position *fro* to position *to*.
+    """
+
+    def __init__(self, car, fro, to):
+        self.car = car
+        self.fro = fro
```

Comment:
Where do you use/call these parameters (fro and to) in the code? If never, do you actually need them?
(Just a minor typo, it's "from" and not "fro", if that's the word you were referring to as I believe :) )